server/src/main/java/io/crate/metadata/GeoReference.java

```diff
@@ -48,7 +40,10 @@
 
 public class GeoReference extends SimpleReference {
 
-    private static final String DEFAULT_TREE = "geohash";
+    public static final String GEOHASH_PREFIX_REE = "geohash";
+    public static final String LEGACY_QUAD_PREFIX_TREE = "legacyquadtree";
+    public static final String QUAD_PREFIX_TREE = "quadtree";
+    public static final String BKD_TREE = "bkdtree";
```

Comment:
Minor: "geohash" and "quadtree" are already defined in `GeoShapeFieldMapper`, we could add new constants there to be consistent with the old code.

Reply:
Done.